packages/react-provider/src/stories/FluentProviderDir.stories.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+import { makeStyles } from '@fluentui/react-make-styles';
+import { FluentProvider } from '@fluentui/react-provider'; // codesandbox-dependency: @fluentui/react-provider ^9.0.0-beta
+
+const useStyles = makeStyles({
+  example: {
+    margin: '5px',
+  },
+  darkThemeBackground: theme => ({
+    backgroundColor: theme.colorBrandBackground,
+  }),
```

Comment:
This is not used in this example 🙄

---

I made few style tweaks to make it looking nicer in docs & canvas.

![image](https://user-images.githubusercontent.com/14183168/139834848-51506a46-d166-492d-928a-f6c4fb8b9320.png)

```tsx
const useStyles = makeStyles({
  example: {
    display: 'flex',
    flexDirection: 'column',
    gap: '5px',
    width: '300px',
  },
  text: theme => ({
    backgroundColor: theme.colorBrandBackground2,
    color: theme.colorBrandForeground2,
    fontSize: '18px',
    border: '1px',
    borderRadius: '5px',
    padding: '5px',
  }),
});

export const Dir = () => {
  const styles = useStyles();

  return (
    <div className={styles.example}>
      <FluentProvider>
        <div className={styles.text}>Text left to right</div>
      </FluentProvider>
      <FluentProvider dir="rtl">
        <div className={styles.text}>نص من اليمين إلى اليسار</div>
      </FluentProvider>
    </div>
  );
};
```